### PR TITLE
Better document level of support for our simulators

### PR DESCRIPTION
**Pull request by imphil**

### CODE REVIEW

**Comment by imphil** [comment]
This isn't meant to be the best ever way of presenting that information, but a small incremental improvement. I have some ideas how to do much better, but that'll be for another day.

File: documentation/source/simulator_support.rst [hunk 1]
```diff
@@ -73,29 +79,30 @@ Issues for this simulator
 Verilator
 =========
 
+.. warning::
+
+    Verilator is in the process of adding more functionality to its VPI interface, which is used by cocotb to access the design.
+    Therefore, Verilator support in cocot is currently experimental.
```
Comment (cmarqu) [inline documentation/source/simulator_support.rst:85]:
```suggestion
    Therefore, Verilator support in cocotb is currently experimental.
```

File: documentation/source/simulator_support.rst [hunk 2]
```diff
@@ -242,12 +249,13 @@ Issues for this simulator
 -------------------------
 
 * `All issues with label category:simulators:questa <https://github.com/cocotb/cocotb/issues?q=is%3Aissue+-label%3Astatus%3Aduplicate+label%3Acategory%3Asimulators%3Aquesta>`_
-
+* Questa 2021.1 and later start to support the VHPI interface in addition to the proprietary FLI interface.
+  However, cocotb only supports the use of Questa through FLI.
```
Comment (ktbarrett) [inline documentation/source/simulator_support.rst:253]: No longer true as of #2408. I should add a newsfragment, which I forgot in the PR. It's now more "they don't support us".

File: documentation/source/simulator_support.rst [hunk 3]
```diff
@@ -327,6 +335,10 @@ Issues for this simulator
 GHDL
 ====
 
+.. warning::
```
Comment (marlonjames) [inline documentation/source/simulator_support.rst:338]: Remove similar warning below.

File: documentation/source/simulator_support.rst (hunk 2 above)
Comment (imphil) [reply documentation/source/simulator_support.rst]: I clarified this a bit. I really want to encourage users to continue using FLI until VHPI support is more stable and tested.

**Comment by imphil** [comment]
Thank for your reviews, updated.

**Review by ktbarrett — APPROVED** [review]

**Pull request merged**